scripts/qemu-dm-wrapper

```diff
@@ -19,6 +19,8 @@ path = "/sbin:/usr/sbin:/bin:/usr/bin"
 
 xenstore_write_cmd = "xenstore-write"
 
+cgroup_slice = ''
```

Comment:
`cgroup_slice` is only set here. How does it every become something other than `''`?